pkgs/development/python-modules/blessings/default.nix

```diff
@@ -3,33 +3,35 @@
   buildPythonPackage,
   fetchPypi,
   six,
-  nose,
+  pynose,
```

Comment:
`pynose` violates the license and will be removed in https://github.com/NixOS/nixpkgs/pull/325669
If you need `nose` for testing, please disable tests in 3.12.

Reply:
Will remove pynose from all applicable packages.